ui/v2.5/src/docs/en/Scraping.md

```diff
@@ -85,7 +85,7 @@ script:
   - query
 ```
 
-This configuration would execute `python iafdScrape.py query`.
+Stash will find the correct python executable for your system, either `python` or `python3`. So for example. this configuration could execute `python iafdScrape.py query` or `python3 iafdScrape.py query`.
```

Comment:
This should be more explicit about what it does. From what I understand of `findPythonExecutable`, this will search for `python3` first, then fall back to use `python` if the former isn't found.